### DISCUSSION TITLE
Conversation example

### DISCUSSION THREAD

**Discussion by sej69** [shinyorg/shiny#1212] [Q&A]
I'm circling back from last summer working on a pet project using the speech recognition engine.  I re-downloaded the samples and am able to compile and deploy to my attached android devices as well as the emulator.

However, when I select the "conversation" menu option and start the conversation, it won't pick up anything in the voice.

I did notice that it's not asking for Microphone permissions so I'm wondering if that is my issue?  I added in 

    <uses-permission android:name="android.permission.MICROPHONE" />

Into the AndroidManifest.xml file but I still don't get a popup.

Any ideas what may be going on?

**Comment by aritchie** [comment]
As I said then, I don't maintain the samples at https://github.com/shinyorg/shiny any longer and it likely needs updating.  The permission required for android speech recognizer is not microphone, it is RECORD_AUDIO as per the android docs: https://developer.android.com/reference/android/speech/SpeechRecognizer

**Comment by sej69** [comment]
As I work on this, is there a way I can update documentation?
